### PR TITLE
Bug Cylance - return array entry as DBot type

### PR DESCRIPTION
**Pull request by MLainer1**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/46204

## Description
return array entry as DBot type

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/Cylance_Protect/Integrations/Cylance_Protect_v2/Cylance_Protect_v2.py [hunk 1]
```diff
@@ -766,18 +775,12 @@ def get_threats():
...
             dbot_score = translate_score(threat['cylance_score'], int(threshold))
-        dbot_score_array.append({
-            'Indicator': threat.get('sha256'),
-            'Type': 'file',
-            'Vendor': 'Cylance Protect',
-            'Score': dbot_score
-        })
-    context_threat = createContext(data=threats, keyTransform=underscoreToCamelCase, removeNull=True)
-    context_threat = add_capitalized_hash_to_context(context_threat)
-    ec = {
```
Comment (jochman) [inline Packs/Cylance_Protect/Integrations/Cylance_Protect_v2/Cylance_Protect_v2.py:777]: get back the file. (You can use `Common.File`)

**Review by jochman — CHANGES_REQUESTED** [review]
Cool.

**Review by jochman — APPROVED** [review]
Nice. 🏞️

File: Packs/Cylance_Protect/ReleaseNotes/1_0_9.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Cylance Protect v2
+- Fixed an issue where multiple DBotScore objects of different indicators returned at once.
```
Comment (avidan-H) [inline Packs/Cylance_Protect/ReleaseNotes/1_0_9.md:4]:
```suggestion
- Fixed an issue where multiple indicators returned DBotScore objects simultaneously.
```

File: Packs/Cylance_Protect/Integrations/Cylance_Protect_v2/Cylance_Protect_v2.py [hunk 3]
```diff
@@ -377,17 +381,18 @@ def get_device_by_hostname():
...
-    }
-    uri = '%s/%s' % (URI_HOSTNAME, hostname)
-    res = api_call(uri=uri, method='get', headers=headers)
-    if not res:
-        return None
-    return res[0]
+def get_hostname_request(hostname):  # pragma: no cover
+
+    access_token = get_authentication_token(scope=SCOPE_DEVICE_READ)  # pragma: no cover
+    headers = {  # pragma: no cover
```
Comment (jochman) [inline Packs/Cylance_Protect/Integrations/Cylance_Protect_v2/Cylance_Protect_v2.py:387]: Change the headers to one call, maybe push it into `api_call`

**Review by jochman — CHANGES_REQUESTED** [review]

**Review by jochman — APPROVED** [review]
🌻

**Pull request merged**